study-builder/studies/pancan/patches/unique-cancer-validation.conf

```diff
@@ -0,0 +1,22 @@
+{
+  "studyGuid": "cmi-pancan",
+  "questionStableIds": "PRIMARY_CANCER_LIST_SELF,PRIMARY_CANCER_LIST_ADD_CHILD",
```

Comment:
Specified a list of composite questions where to add UNIQUE rule:
`"PRIMARY_CANCER_LIST_SELF,PRIMARY_CANCER_LIST_ADD_CHILD"`

Reply:
How about "PRIMARY_CANCER_LIST_CHILD" . I believe we need validation for that list too.

Reply:
Fixed